app/controllers/pxe_controller.rb

```diff
@@ -217,7 +217,7 @@ def replace_right_cell(nodetype, replace_trees = [])
           end
         # resetting ManageIQ.oneTransition.oneTrans when tab loads
         presenter[:extra_js] << 'ManageIQ.oneTransition.oneTrans = 0;'
-        if %(save reset).include?(params[:button]) && is_browser_ie?
+        if params[:button].present? && %(save reset).include?(params[:button]) && is_browser_ie?
```

Comment:
This should have worked before.  `include?` woud handle nils just fine.  Do you have a spec exhibiting the error before and fixed after?